sparql/sparql.go

```diff
@@ -9,8 +9,9 @@ type Sparql struct {
 
 // Query is a struct to hold a full SPARQL query
 type Query struct {
-	Parts    []Sparql
-	Prefixes map[string]string
+	Parts      []Sparql
+	Prefixes   map[string]string
+	NamedGraph string
```

Comment:
Could this be `*string` so that it would permit a NULL value rather than `""`?

Reply:
Sure.